src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -33330,5 +33330,36 @@ public class C5 : I1<C5>
                     );
             }
         }
+
+        [Fact, WorkItem(65685, "https://github.com/dotnet/roslyn/issues/65685")]
+        public void NoDuplicateConversionForImplicitAndExplicit()
+        {
+            CreateCompilation("""
+                interface ISam1<T> where T : ISam1<T>
+                {
+                    static abstract T operator +(T a, T b);
```

Comment:
>static abstract T operator +(T a, T b); [](http://example.com/codeflow?start=20&length=39)

For completeness, let's also add and verify a lack of conflict for:
- a unary operator
- an explicit conversion operator
- an explicit checked operator